src/entities/gameDetails/model/slice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice, isPending, isRejected } from '@reduxjs/toolkit';
+import { getGameDetails } from '../api';
+import { TGameDetails } from '../../../types';
+
+type TGameDetailsState = {
+  details: TGameDetails | null;
```

Comment:
точно нужен null? Может просто:
```suggestion
  details?: TGameDetails;
```

Reply:
Fixed